conf/config-default.yaml

```diff
@@ -329,7 +329,8 @@ plugins:                          # plugin list (sorted by priority)
   - request-id                     # priority: 11010
   - fault-injection                # priority: 11000
   - serverless-pre-function        # priority: 10000
-  #- batch-requests                # priority: 4010
+  - mocking                        # priority: 9900
+#  - batch-requests                 # priority: 4010
```

Comment:
Please note the indentation. And maybe we should put it together with `fault-injection` instead of `serverless-pre-function`?

Reply:
> Please note the indentation. And maybe we should put it together with `fault-injection` instead of `serverless-pre-function`?
ok,it's a good suggestion

Reply:
Please sort the list by priority